src/main/java/com/epam/ta/reportportal/database/entity/LogLevel.java

```diff
@@ -63,12 +65,8 @@ public boolean isGreaterOrEqual(LogLevel r) {
 	 * Convert the string passed as argument to a Level. If there are no such level throws exception
 	 */
 	public static LogLevel toLevel(String levelString) {
-		for (LogLevel level : LogLevel.values()) {
-			if (level.name().equalsIgnoreCase(levelString)) {
-				return level;
-			}
-		}
-		return null;
+		return Arrays.stream(LogLevel.values()).filter(level -> level.name()
```

Comment:
Since we use this operation too often, i would suggest to make static map <levelString,Level> and just convert this method to get from map